### PR TITLE
[fix] don't watch `outDir` except `generated`

### PR DESCRIPTION
**Pull request by benmccann**
Fixes error on Windows CI

### CODE REVIEW

**Comment by Conduitry** [comment]
:( `Error: EPERM: operation not permitted, lstat '\\?\D:\a\kit\kit\packages\kit\test\apps\amp\.svelte-kit\types\src\routes'` Huh

File: .changeset/eight-tomatoes-yawn.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+[fix] don't watch .svelte-kit directory
```
Comment (benmccann) [inline .changeset/eight-tomatoes-yawn.md:5]:
```suggestion
[fix] don't watch generated types directory
```

**Comment by Rich-Harris** [comment]
Nice! Any theory as to why ignoring the entire `outDir` doesn't work?

**Comment by Rich-Harris** [comment]
Okay, looking at the failures more closely (after you fixed `exclude` to `ignored` in 853d9826915e61a14dd18d90309104bef2523864), they were just flakiness rather than a 'proper' failure.

I think what we want to do is ignore the entire `outDir` except for the `generated` directory (I tried ignoring the entire `outDir`, but then it doesn't pick up changes to the route manifest, e.g. if you navigate to `/unknown` and _then_ created `src/routes/unknown.svelte`, you need to watch `generated` or it won't refresh to that page)

File: .changeset/eight-tomatoes-yawn.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+[fix] don't watch generated types directory
```
Comment (Conduitry) [inline .changeset/eight-tomatoes-yawn.md:5]: The changeset police is here to remind you to update your changeset before you jubilantly/exhaustedly merge this.

Comment (Rich-Harris) [inline .changeset/eight-tomatoes-yawn.md:5]: THANKS DAD

```suggestion
[fix] don't watch `outDir`, except for the `generated` directory
```

**Review by Rich-Harris — APPROVED** [review]

**Pull request merged**